ci/checks/style.sh

```diff
@@ -82,4 +82,21 @@ else
   echo -e "\n\n>>>> PASSED: clang format check\n\n"
 fi
 
+# Check for a good doxygen comments in the c++ base
+DOXY_DIR=doxygen_check
+DOXYGEN=`(mkdir -p $DOXY_DIR && cd $DOXY_DIR && cmake ../cpp && make doc && rm -rf $DOXY_DIR) 2>&1`
```

Comment:
The style checker job is not really meant for building any code. You should move to to either cpu or gpu builds.

Reply:
Thanks @raydouglass . I have moved this inside `ci/gpu/build.sh`. Please re-review.